Gemfile

```diff
@@ -12,6 +12,7 @@ else
   raise "Unsupported Ruby on Rails version configured in settings.yaml: #{SETTINGS[:rails]}"
 end
 
+gem 'rake', '< 13.0.4'
```

Comment:
Please add a comment on why this is pinned so we remember to remove once it's fixed